guide/14-deep-learning/how_retinanet_works.ipynb

```diff
@@ -0,0 +1,170 @@
+{
```

Comment:
<p>Since the defaults are being used, we don't need to specify them, and can create RetineNet using:</p><p><code>model = RetinaNet(data)</code></p><p> </p><p>If you want to show how the defaults can be changed, I suggest using simpler numbers other than "2^0, 2^⅓,2^⅔"</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/562/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>